_posts/2021-01-18-changelog-139.markdown

```diff
@@ -0,0 +1,115 @@
+---
+layout: post
+title: "IntelliJ Rust Changelog #139"
+date: 2021-01-18 17:00:00 +0300
+---
+
+
+## New Features
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/6497 -->
+* Take into account attributes under `#[cfg_attr]` attribute in our type inference, name resolution, and other analysis.
```

Comment:
```suggestion
* Take into account attributes under [`cfg_attr`](https://doc.rust-lang.org/reference/conditional-compilation.html#the-cfg_attr-attribute) attribute during type inference, name resolution, and other analysis.
```

I prefer to use attribute name without `#[]` in descriptions because attributes can be nested